### ISSUE TITLE
Apps verlansamen SocialEngine

### ISSUE THREAD

**Issue by KlaudiaSt** [timemb/herzenspeople#22]
Herr Damm von Heidelpay sagte mir, dass das Programm, je mehr Apps hinzukommen, langsamer wird.
Aber es gäbe eine Firma, die das Programm wieder schneller macht. 
Darüber solltest du mit Ihm sprechen, er kennt sich mit SocialEngine aus, da er wohl selbst ne Zeit lang einen Blog darüber betrieben hat.

**Comment by timemb** [comment]
Danke für den Input. Derzeit weiß ich noch nicht wie schnell die SocialEngine sein wird.

Wir haben noch nicht alle Plugins installiert. Manche werden auch nicht an jeder Stelle gebraucht.
Wir befinden uns auch noch im Entwicklungsmodus, der ist zusätzlich Langsamer da er den Browser zwingt alle Dateien bei Seitenaufruf neu zu laden, um Änderungen immer direkt zu sehen.

Wenn alle Grundfunktionen eingestellt sind sollten wir das beim Testen der Community vor Start auf jeden Fall bedenken. Hat er gesagt welche Firma das war?

**Comment by KlaudiaSt** [comment]
Nein, da habe ich ihn unterbrochen und gesagt, dass du dich bei ihm melden wirst. Ist doch sinnvoller, wenn er dir alles erzählt, was er so weiss.
Er hat wohl auch seinerzeit die App für Heidelpay für 800 Dollar bei younetco programmieren lassen.
